src/foam/nanos/notification/Notification.js

```diff
@@ -197,14 +197,14 @@ foam.CLASS({
       name: 'authorizeOnCreate',
       javaCode: `
       AuthService auth = (AuthService) x.get("auth");
-      if ( ! checkOwnership(x) && ! auth.check(x, createPermission("create")) ) throw new AuthorizationException("You don't have permission to create this notification.");
+      if ( ! checkOwnership(x) && ! auth.check(x, createPermission("create")) && ! getTransient() ) throw new AuthorizationException("You don't have permission to create this notification.");
```

Comment:
Objects can be transient?

Reply:
@zahnno This is a 'model' property on Notification. Not related to property property 'transient'.

Reply:
Ahh I see it now. What's the purpose of it?